aii-opennebula/src/main/resources/vmtemplate.tt

```diff
@@ -69,4 +69,22 @@ DATA = "<vcpu>[% vcpus %]</vcpu><cpu mode='host-passthrough'><topology sockets='
 TYPE = "kvm"
 [%-    END %]
 ]
+[%- FOREACH item IN system.opennebula.pci %]
+PCI = [
+[%    FILTER indent -%]
+[%    INCLUDE 'aii-opennebula/pci_level1.tt' 
+               data=item -%]
+[%-   END -%]
+]
+[%- END %]
+[%- FOR pair IN hardware.cards.ib.pairs %]
```

Comment:
@stdweird I was thinking about this, maybe we should keep both. I mean, if you have already the IB info with the PCI address in your quattor db the info will be used automatically and inserted in your VM template, but you can also use `/system/opennebula/pci` to insert extra cards that are not included in your db or for testing purposes, if you want to test a GPU for example. What do you think?